app/src/test/java/io/intrepid/contest/screens/contestcreation/describecontest/DescribeContestPresenterTest.java

```diff
@@ -25,18 +25,23 @@ public void setup() {
                                                                 TestPresenterConfiguration.createTestConfiguration());
     }
 
-    @Test
-    public void emptyContestDescriptionShouldYieldError() {
-        String EMPTY_TEXT = "";
-        describeContestPresenter.onNextClicked(EMPTY_TEXT);
-        verify(mockView).showError();
-    }
-
     @After
     public void validContestDescriptionShouldYieldError() {
```

Comment:
good catch! And for the name, I'll name it with the format we discussed "onNextClickShouldTriggerViewToSaveContestDescription()"